editor/gui/editor_spin_slider.cpp

```diff
@@ -69,6 +70,7 @@ void EditorSpinSlider::gui_input(const Ref<InputEvent> &p_event) {
 				} else {
 					grabbing_spinner_attempt = true;
 					grabbing_spinner_dist_cache = 0;
+					grabbing_spinner_speed = EditorSettings::get_singleton()->get("interface/inspector/float_drag_speed");
```

Comment:
Fetching the editor setting every time there's a click event seems a bit wasteful, that's a lot of processing for something that's not going to change often.

I would move this at least to a member variable set in the constructor, so it's fetched only once per slider.

Reply:
makes sense, done!